avocado_vt/loader.py

```diff
@@ -199,7 +199,9 @@ def discover(self, url, which_tests=loader.DiscoverMode.DEFAULT):
         for params in (_ for _ in cartesian_parser.get_dicts()):
             # Evaluate the proper avocado-vt test name
             test_name = None
-            if get_opt(self.config, 'vt.type') == "spice":
+            if get_opt(self.config, 'vt.short_names_when_config'):
```

Comment:
Better to add one more expression `get_opt(self.config, 'vt.config')` if the option works only if using `--vt-config`, otherwise, I suggest introducing the option without the suffix `-when-config`.

Reply:
This option is not very useful for usage without config file since users can customize guests, machine_types, etc. in `avocado run` command. I will add `vt.config` together.